packages/marko-web-p1-events/components/track-scroll-depth/content.marko

```diff
@@ -0,0 +1,47 @@
+import { getAsArray } from "@parameter1/base-cms-object-path";
+import ns from "../../utils/create-namespace";
+
+$ const { site } = out.global;
+$ const {
+  content,
+  action,
+  category = 'Content',
```

Comment:
Don't allow action or category to be customized in the specific components -- content should always be content category and action should always be scroll.

The category should be required on your generic/static component, but action should still be hardcoded to scroll.